distribution/changelog.txt

```diff
@@ -14,6 +14,7 @@
 - Fix: RCT1 scenarios have more items in the object list than are present in the park or the research list.
 - Improved: [#6530] Allow water and land height changes on park borders.
 - Improved: [#11390] Build hash written to screenshot metadata.
+- Feature: [#10572] Cheat to allow building at invalid heights.
```

Comment:
Should be grouped together with the other features.

Reply:
Got it, fixed now.